### ISSUE TITLE
Pretrained Checkpoints

### ISSUE THREAD

**Issue by bilalsal** [NVIDIA/partialconv#23]
First, congratulations to the PartialConv team, it is a fascinating contribution in many ways.

Also, thank you for making pdresnet checkpoints available, and for the insightful comparison with the other padding modes!
Would it also be possible to have pre-trained weights for REFLECT and REPLICATE?
Thank you!
